src/smart-components/group/groups.js

```diff
@@ -81,6 +100,9 @@ const Groups = ({ fetchGroups, isLoading, pagination, history: { push }, groups
             data={ groups }
             createRows={ createRows }
             columns={ columns }
+            isSelectable={ true }
```

Comment:
`isSelectable={true}` is equivalent to `isSelectable`. This should be caught by linter.